images/patroni/entrypoint.sh

```diff
@@ -10,13 +10,26 @@ fi
 cat > /home/postgres/patroni.yml <<__EOF__
 bootstrap:
   dcs:
-    loop_wait: ${PATRONI_LOOP_WAIT}
-    ttl: ${PATRONI_TTL}
-    retry_timeout: ${PATRONI_RETRY_TIMEOUT}
-    master_start_timeout: ${PATRONI_MASTER_START_TIMEOUT}
-    maximum_lag_on_failover: ${PATRONI_MAXIMUM_LAG_ON_FAILOVER}
```

Comment:
Just wonder why you remove those params? Are they no longer supported?

Reply:
When I copy `/entrypoint.sh` from `jberkus/patroni-simple:v13` it's identical to my current one - without changes from this PR...